lib/archive/tar/minitar/reader.rb

```diff
@@ -5,6 +5,7 @@ module Archive::Tar::Minitar
   # stream may be sequential or random access, but certain features only work
   # with random access data streams.
   class Reader
+    include Enumerable
```

Comment:
Right. I know what it does; I’m wondering why you added it because I don’t see the `Reader` as an `Enumerable`; it’s something that sits on top of a stream. That is, unless you can `#rewind` on the stream, the `Reader` is possible for exactly *one* enumeration.